src/containers/common/HomeContainer.jsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { Actions } from 'jumpstate';
+
 import Home from '../../components/common/Home';
 
 class HomeContainer extends React.Component {
   constructor(props) {
     super(props);
   }
 
+  componentDidMount() {
+    // Actions.getPrograms();
```

Comment:
Is this a placeholder for future functionality, or a feature that was in test but no longer used?

Reply:
Yes! Programs still need the show route on the API so GET requests work: https://github.com/zooniverse/education-api/issues/62

I'd like to be able to request for the programs for the home page listing and then for the selectedProgram if a user navigates directly to I2A or Wildcam Darien. The mocks in the programs duck shows what I'm envisioning as the data we get back from the EduAPI.